### PR TITLE
DOCS-7699 Add Edge info

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

**Review by clies212 — DISMISSED** [review]
We can't use Edge in external facing documentation. We need to use this terminology: Inbound traffic is processed using an internal load balancer managed by Anypoint Runtime Fabric

I would think Edge should be replaced with "Internal Load Balancer" or something similar. Can you search and make sure that we are not using Edge anywhere else?

File: modules/ROOT/pages/deploy-resource-allocation.adoc [hunk 1]
```diff
@@ -42,7 +42,7 @@ When using CPU bursting, you must consider the following factors:
 +
 Regardless of what approach is chosen, make sure you run tests to validate performance.
 
-== Memory Allocation
+== Application Memory Allocation
```
Comment (sarjeet2013) [inline modules/ROOT/pages/deploy-resource-allocation.adoc]: As the summary say, this is memory allocation for Mule application or API gateway, so please revert it back to Memory Allocation.

**Review by sarjeet2013 — CHANGES_REQUESTED** [review]
A minor comment, otherwise LGTM.

File: modules/ROOT/pages/deploy-resource-allocation.adoc (hunk 1 above)
Comment (paigehargrave) [reply modules/ROOT/pages/deploy-resource-allocation.adoc]: Thank you - done!

File: modules/ROOT/pages/deploy-resource-allocation.adoc [hunk 2]
```diff
@@ -90,6 +90,19 @@ The resources allocated to your Mule application determine the application's per
 [NOTE]
 Run performance and load testing on your Mule applications to determine the number of resources to allocate.
 
+== Internal Load Balancer Memory Allocation
```
Comment (rddheath) [inline modules/ROOT/pages/deploy-resource-allocation.adoc:93]: What does "Internal" apply to: "load balancer", "memory", "allocation", or "memory allocation"? I don't think what it applies to is clear.

File: modules/ROOT/pages/deploy-resource-allocation.adoc [hunk 3]
```diff
@@ -90,6 +90,19 @@ The resources allocated to your Mule application determine the application's per
...
+
+Internal load balancer memory requirements are impacted by the number of threads, response time latency, and message sizes. The following internal load balancer memory allocations are recommended:
```
Comment (rddheath) [inline modules/ROOT/pages/deploy-resource-allocation.adoc]: So, I guess "internal" refers to "memory"? Then, perhaps the heading above could be "Allocating Memory in an Internal Load Balancer". And in this sentence here, "Internal load balancer memory requirements" could be "Requirements for memory in an internal load balancer".
"Impacted" is too strong a word. How about "affected"?
"response time latency" should be "response-time latency".
How are the requirements affected by these things? You say they are, but then don't explain how, even briefly.
"The following internal load balancer memory allocations are recommended" repeats the long phrase "internal load balancer memory" that began the previous sentence. How about changing the sentence to read like this: "MuleSoft recommends these general guidelines for determining the proper allocation of memory:"?

File: modules/ROOT/pages/deploy-resource-allocation.adoc [hunk 4]
```diff
@@ -90,6 +90,19 @@ The resources allocated to your Mule application determine the application's per
...
+
+* .5 GB (Default): Fewer than 500 simultaneous active connections.
```
Comment (rddheath) [inline modules/ROOT/pages/deploy-resource-allocation.adoc]: Could you recast this as a sentence, such as this one: "Allocate 0.5 GB of memory if the load balancer handles fewer than 500 active connections simultaneously. 0.5 GB is the default allocation."?

File: modules/ROOT/pages/deploy-resource-allocation.adoc [hunk 5]
```diff
@@ -90,6 +90,19 @@ The resources allocated to your Mule application determine the application's per
...
+* 1.5 GB (Large): One or both of the following scenarios:
```
Comment (rddheath) [inline modules/ROOT/pages/deploy-resource-allocation.adoc]: Could you recast this guideline as a sentence, such as this one: "Allocate 1.5 GB of memory if one or both of these conditions are met:
* The load balancer handles 500 or more active connections simultaneously.
* Security policies are enabled for the load balancer."?

File: modules/ROOT/pages/deploy-resource-allocation.adoc [hunk 6]
```diff
@@ -90,6 +90,19 @@ The resources allocated to your Mule application determine the application's per
...
+
+** 500 or more simultaneous active connections.
+** Security policies enabled.
+
+[NOTE]
+These are general guidelines, and individual environments might require adjustments.
```
Comment (rddheath) [inline modules/ROOT/pages/deploy-resource-allocation.adoc:104]: Adjustments to what: guidelines or the amount of memory allocated?
In either case, it doesn't seem as though this sentence is needed.
"General" implies that the guidelines won't fit every case exactly.

**Review by rddheath — APPROVED** [review]

**Review by sarjeet2013 — APPROVED** [review]

**Pull request merged**